src/App.jsx

```diff
@@ -2,6 +2,8 @@ import './App.scss';
 
 import todosFromServer from './api/todos.json';
 import usersFromServer from './api/users.json';
+import { TodoList } from './components/TodoList';
+import { UserInfo } from './components/UserInfo';
```

Comment:
looks like UserInfo not use in this module.
```suggestion
```
![Screenshot from 2023-09-07 12-07-55](https://github.com/mate-academy/react_static-list-of-todos-js/assets/33029864/ac5ae4a2-2792-4a7e-812b-2b29cbe88416)